src/main/resources/application-dev.yaml

```diff
@@ -6,7 +6,7 @@ spring:
     password: "pass"
   datasource:
```

Comment:
consolidate most of these setting in application.yaml and just override them in specific profiles

Reply:
done